modules/cassandra/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>testcontainers-parent</artifactId>
+        <version>0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>cassandra</artifactId>
+    <name>TestContainers :: Cassandra</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>jdbc</artifactId>
```

Comment:
Cassandra (and your implementation) does not provide a JDBC support at all, we should remove the dependency on JDBC module. 
`AbstractDatabaseDelegate` seems to be over-engineered as well

Reply:
Good point. I separated Script engine from SQL implementation, but didn't do from modules perspective. I created new commons module that contains ScriptUtils (which is not SQL-tied up and can run any scripts) and DatabaseDelegate interface with its abstract child. Implementations of the delegate lie in corresponding modules.
Cassamdra module now also depends only on commons module.